pandas/tests/frame/conftest.py

```diff
@@ -1,10 +1,17 @@
+from itertools import product
+
 import numpy as np
 import pytest
 
 from pandas import DataFrame, NaT, date_range
 import pandas._testing as tm
 
 
+@pytest.fixture(params=product([True, False], [True, False]))
+def close_open_fixture(request):
+    return request.param
```

Comment:
if this does not have utility beyond the one test in pandas/tests/frame/methods/test_between_time.py, I would remove in a follow-up

Reply:
agreed